OptimizelySDKCore/OptimizelySDKCore/Optimizely.m

```diff
@@ -112,52 +112,61 @@ - (OPTLYVariation *)activate:(NSString *)experimentKey
                   attributes:(NSDictionary<NSString *,NSString *> *)attributes
                     callback:(void (^)(NSError *))callback {
     
-    // get variation
-    OPTLYVariation *variation = [self variation:experimentKey userId:userId attributes:attributes];
+    __weak void (^_callback)(NSError *) = callback ? : ^(NSError *error) {};
+    
+    if ([Optimizely isEmptyString:experimentKey]) {
```

Comment:
I'd rather we make this into another PR to keep this one from being longer than it already is

Reply:
These input validations were missing from Activate and Track APIs